### ISSUE TITLE
Add QRcode reader

### ISSUE THREAD

**Issue by sheychen290** [SimpleMobileTools/Simple-Camera#73]
It's so usefull and logic (IMO) to place a qrcode button in a camera app.
Thank

**Comment by F3000** [comment]
it would be greate when it is like the ios11 feature ... open Camera -> hold camera over qr-code -> app display a little pop-up window with the informations decrypted in the code ->user can click to open it (e.g. a link)

**Comment by Klimmbimmel** [comment]
Nah, not needed, other things are more important. Just use Firefox with its integrated qr-button

**Comment by thomas-mc-work** [comment]
@Klimmbimmel There are other use cases like reading WI-FI credentials that Firefox cannot satisfy.

**Comment by Pyromane** [comment]
That would be a great benefit!

**Comment by inson1** [comment]
@Pyromane SMT apps were sold to ad company. https://github.com/SimpleMobileTools/General-Discussion/issues/241
Fork is here https://github.com/FossifyOrg

**Comment by Pyromane** [comment]
> @Pyromane SMT apps were sold to ad company. [SimpleMobileTools/General-Discussion#241](https://github.com/SimpleMobileTools/General-Discussion/issues/241) Fork is here https://github.com/FossifyOrg

Thank you for the important information!
